### PR TITLE
luminous: librgw: initialize curl and http client for multisite

### PR DESCRIPTION
**Pull request by theanalyst**
http://tracker.ceph.com/issues/36414

### CODE REVIEW

File: src/rgw/librgw.cc [hunk 1]
```diff
@@ -489,6 +491,8 @@ namespace rgw {
     rgw_tools_init(g_ceph_context);
 
     rgw_init_resolver();
+    rgw::curl::setup_curl(boost::none);
+    rgw_http_client_init(g_ceph_context);
```
Comment (theanalyst) [inline src/rgw/librgw.cc]: this must be dropped, as this was mimic introduced, waiting for jenkins to confirm

Comment (smithfarm) [reply src/rgw/librgw.cc]: does not pass "make check" - @theanalyst Is that the confirmation you were looking for?

**Comment by cbodley** [comment]
this backport was resolved in https://github.com/ceph/ceph/pull/24909

**Comment by smithfarm** [comment]
Sorry @theanalyst I forgot you had this open.

**Pull request closed without merging**